When I started to prepare for the regression tests I found this warning.
Not crucial but for completeness we might want to fix it.
```
src/org/daisy/dotify/formatter/impl/search/VolumeKeepPriority.java:122: warning: no @return
        public static Comparator<VolumeKeepPriority> naturalOrder() {
```

--- a/src/org/daisy/dotify/formatter/impl/search/VolumeKeepPriority.java
+++ b/src/org/daisy/dotify/formatter/impl/search/VolumeKeepPriority.java
@@ -118,6 +118,9 @@ public final class VolumeKeepPriority implements Comparable<VolumeKeepPriority> 
 	 * <p>Returns a comparator that imposes the natural ordering on {@link VolumeKeepPriority} objects.</p>
 	 *
 	 * <p>Objects are ordered according to {@link #compareTo(VolumeKeepPriority)}, from lower to higher priority.</p>
+	 *
+	 * @return a negative integer, zero, or a positive integer as the first argument is less
+	 *         than, equal to, or greater than the second.
 	 */
 	public static Comparator<VolumeKeepPriority> naturalOrder() {
 		if (comparator == null) {

--- a/src/org/daisy/dotify/formatter/impl/search/VolumeKeepPriority.java
+++ b/src/org/daisy/dotify/formatter/impl/search/VolumeKeepPriority.java
@@ -119,8 +119,7 @@ public final class VolumeKeepPriority implements Comparable<VolumeKeepPriority> 
 	 *
 	 * <p>Objects are ordered according to {@link #compareTo(VolumeKeepPriority)}, from lower to higher priority.</p>
 	 *
-	 * @return a negative integer, zero, or a positive integer as the first argument is less
-	 *         than, equal to, or greater than the second.
+	 * @return a comparator that imposes the natural ordering on {@link VolumeKeepPriority} objects.
 	 */
 	public static Comparator<VolumeKeepPriority> naturalOrder() {
 		if (comparator == null) {

